backtrace-library/src/main/java/backtraceio/library/BacktraceClient.java

```diff
@@ -24,6 +24,28 @@ public class BacktraceClient extends BacktraceBase {
      */
     private BacktraceANRWatchdog anrWatchdog;
 
+    /**
+     * Enables Crash loop detection to verify if the application is in the crash loop.
+     */
+    public static boolean EnableCrashLoopDetection() {
+        return BacktraceDatabase.EnableCrashLoopDetection();
+    }
+
+    /**
+    * Determine if application requires a safe mode to launch.
+    */
+    public static boolean IsSafeModeRequired() {
+        return BacktraceDatabase.IsSafeModeRequired();
+    }
+    
+    /**
+     * Returns the number of consecutive application crashes 
+     */
+    public static int ConsecutiveCrashesCount() {
+        return BacktraceDatabase.ConsecutiveCrashesCount();
+    }
+
+    
```

Comment:
I'm not sure I like one more layer in the overall call stack to CLD methods in the crashpad, it is 5th seems :)
But having it under Client is still good.

Reply:
Please take a look into the pull request description - most customers will use client instead of the database object. It's easier and it's less to learn for them